### DISCUSSION TITLE
Community meeting January 14th 2022 (v1.45)

### DISCUSSION THREAD

**Discussion by lucasponce** [kiali/kiali#4651] [Community meetings]
Hi Kiali & Istio users,

Next Friday we will continue with the Kiali community meetings in the following format:

spiral_notepad Agenda

Kiali-wide announcements: A place to announce Kiali news
Sprint / release demo: Space to showcase the work introduced in the next release
Submitted topics: Place your questions, topics, thoughts, agenda items in the discussion below and we'll approach them.
Open Q&A: Space for remaining topics, questions, thoughts not covered in this discussion.
microphone Add a topic to the agenda

Place a comment in this discussion with the topic you would like the community (contributors, maintainers, users) discuss during the meeting. In the Submitted topics section we'll go through all the topics and will open the mics to approach it.
We encourage you to join the meeting in order to get a complete answer and get as far as we can.

information_source Community meeting info

calendar Friday January 14th, 2022
clock1030 10am (EST) / 4pm (CET) / 11pm (HKT)
computer https://bluejeans.com/7625852117

spiral_calendar Subscribe to Kiali Calendar

Subscribe to our calendar to stay tuned about working meetings, community meetings, and other non-spammy events:
Subscribe now! arrow_right https://bit.ly/kiali-calendar

**Comment by Xunzhuo** [comment]
Ops... It seems like I missed the time. Maybe I should subscribe the calendar
